tests/test_http_server_cb/test_http_server_cb.cpp

```diff
@@ -194,22 +194,51 @@ gw_status_resume_http_relaying(void)
 http_server_resp_t
 http_check_post_advs(const http_check_params_t* const p_params, const TimeUnitsSeconds_t timeout_seconds)
 {
-    const http_server_resp_t resp = {
-        .http_resp_code       = HTTP_RESP_CODE_500,
-        .content_location     = HTTP_CONTENT_LOCATION_NO_CONTENT,
-        .flag_no_cache        = true,
-        .flag_add_header_date = true,
-        .content_type         = HTTP_CONENT_TYPE_TEXT_HTML,
-        .p_content_type_param = NULL,
-        .content_len          = 0,
-        .content_encoding     = HTTP_CONENT_ENCODING_NONE,
-        .select_location = {
-            .memory = {
-                .p_buf = NULL,
+    if ((0 == strcmp(p_params->p_url, "https://myserver.com/")) && (GW_CFG_HTTP_AUTH_TYPE_NONE == p_params->auth_type))
+    {
+        static const char resp_content[]
+            = "{\n"
+              "\t\"status\":\t200,\n"
+              "\t\"message\":\t\"{}\"\n"
+              "}";
+        const char* p_resp = strdup(resp_content);
+        assert(NULL != p_resp);
+        const http_server_resp_t resp = {
+            .http_resp_code       = HTTP_RESP_CODE_200,
+            .content_location     = HTTP_CONTENT_LOCATION_HEAP,
+            .flag_no_cache        = true,
+            .flag_add_header_date = true,
+            .content_type         = HTTP_CONENT_TYPE_APPLICATION_JSON,
```

Comment:
This should be HTTP_CONTENT_TYPE_APPLICATION_JSON, there's a type CONENT

Reply:
> This should be HTTP_CONTENT_TYPE_APPLICATION_JSON, there's a type CONENT

Thanks, it requires fixing it in a separate component - esp32-wifi-manager, I'll do it in a separate PR.